pynucastro/nucdata/spin_table.py

```diff
@@ -81,22 +57,20 @@ def _read_table(self):
             A = int(ls.pop(0))
             Z = int(ls.pop(0))
             ls.pop(0)           # Spin
-            spin_states = float(ls.pop(0))
+            spin_states = int(ls.pop(0))
```

Comment:
I suggest to keep this number as a float because you will be using it in operations with np.float variables

Reply:
Isn't it always an integer though? numpy will automatically cast things to the right data type when it needs to.

Reply:
mmm, I leave you on your discretion but mixing types is something I wouldn't do